cmd/aro/mirror.go

```diff
@@ -99,9 +99,19 @@ func mirror(ctx context.Context, log *logrus.Entry) error {
 					Payload: version.InstallStream.PullSpec,
 				})
 			} else {
+				vers, err := version.ParseVersion(arg)
+				if err != nil {
+					return nil
```

Comment:
Shouldn't we return `err` instead of `nil`?

Reply:
good catch - ty